doc/install/dependencies.rst

```diff
@@ -79,45 +79,32 @@ Redhat linux users can do likewise with:
 MOAB installation
 ~~~~~~~~~~~~~~~~~
 
-As of DAGMC version 3.1, MOAB version 5.1.0 or higher is required. The following
+As of DAGMC version 3.2, a MOAB version between 5.1.0 and 5.3.0 is required. The following
```

Comment:
Why does this need to be between `5.10` and `5.3.0` again? I thought we could also support newer versions.

Reply:
There are significant changes in v5.4.0 that are causing some [build fail](https://github.com/svalinn/DAGMC/pull/860#issuecomment-1541352060). We need to resolve this issue before proceeding to v5.4 MOAB.

Reply:
Ah, I see. Perhaps we could clarify that the restriction to 5.4.0 is only for Windows if that is the case?

Reply:
Our CI is stuck on 5.3.0 until we can figure out all the issues.  It's a limitation for Ubuntu as well for 20.04 and older - seems to work on 22.04, though.

With all those caveats, it's seems best to stick with 5.3.0 for now

Reply:
Fair enough! Strange though... hopefully we can figure it out someday.